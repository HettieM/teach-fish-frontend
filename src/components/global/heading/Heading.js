import styled from "styled-components";
import {
  bgColour,
  fontColour,
  primaryColour,
  secondaryColour,
  accentColour,
  linkColour,
} from "../colours";

const Heading = styled.h1`
  font-family: "Josefin Sans", sans-serif;
  font-size: 2rem;
  color: ${fontColour};

  @media (min-width: 600px) {
    font-size: 4rem;
  
`;

export { Heading };
