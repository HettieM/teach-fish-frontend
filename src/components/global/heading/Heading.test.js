import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import { Heading } from "./Heading";

test("Heading component is styled", () => {
  const tree = renderer.create(<Heading />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule("color", "navy");
  expect(tree).toHaveStyleRule("font-size", "2rem");
});
