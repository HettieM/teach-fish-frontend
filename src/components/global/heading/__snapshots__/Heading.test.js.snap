// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading component is styled 1`] = `
.c0 {
  font-family: "Josefin Sans",sans-serif;
  font-size: 2rem;
  color: navy;
}

<h1
  className="c0"
/>
`;
